src/main/java/com/linkedin/kmf/common/Utils.java

```diff
@@ -150,7 +150,7 @@ public static GenericRecord genericRecordFromJson(String message) {
     return record;
   }
 
```

Comment:
@smccauliff merging this PR as this is a small change that unblocks internal MP testing.